airflow/configuration.py

```diff
@@ -142,7 +142,10 @@ def run_command(command):
     },
     'github_enterprise': {
         'api_rev': 'v3'
-    }
+    },
+    'admin': {
+        'hide_confidential_ui_fields': True,
+    },
 }
```

Comment:
@criccomini updated the logic, it should check if any of the `key_name` contains any words in the confidential_fields list
<img width="774" alt="screen shot 2016-05-20 at 3 57 19 pm" src="https://cloud.githubusercontent.com/assets/831105/15444264/8f2c62ec-1ea3-11e6-9040-17aaec51699c.png">